bundles/AdminBundle/Resources/public/js/pimcore/object/classes/class.js

```diff
@@ -1083,22 +1083,28 @@ pimcore.object.classes.klass = Class.create({
             value: data.index_key
         };
 
+        //fixes data to match store model
+        var indexesArray = [];
+        Object.values(data.index_columns).forEach(column => {
```

Comment:
Will probably need to do a `null` check here.

![image](https://user-images.githubusercontent.com/58705994/210445388-700518ea-fffb-454c-96b1-79c6f31e37f0.png)